Makefile

```diff
@@ -52,7 +52,7 @@ endef
 # Build
 # ------------------------------------------------------------------------------
 
-CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false,allowDangerousTypes=true"
+CRD_OPTIONS ?= "+crd:allowDangerousTypes=true"
```

Comment:
No apparent good reason, it looks like they were only useful for K8S versions we didn't support to begin with:

https://book.kubebuilder.io/reference/controller-gen.html?highlight=trivial#generators
> Single “trivial-version“ CRDs are compatible with older (pre 1.13) Kubernetes API servers. The storage version‘s schema will be used as the CRD‘s schema. Only works with the v1beta1 CRD version.

0.7 doesn't support it anymore since it only supports v1 CRDs.